sycamore/scans/file_scan.py

```diff
@@ -128,6 +128,8 @@ def _to_document(self, dict: dict[str, Any]) -> dict[str, bytes]:
         if self._is_s3_scheme():
             dict["path"] = "s3://" + dict["path"]
         properties.update({"path": dict["path"]})
+        if "filetype" not in properties and self._binary_format is not None:
+            properties["filetype"] = self._binary_format
```

Comment:
To match what was observed on the PDF side, the value would have to be 'text/html'.  We do want to be consistent in using MIME.  To do that without being hacky may be a bit harder.

Reply:
Made it consistent with MIME by translating the three formats we handle in the partitioner.